src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -106,7 +106,50 @@ private void updateHearing(HearingWrapper wrapper) throws UpdateCaseException, I
                 response.toString());
 
         hearingResponseUpdate(wrapper, response);
+    }
+
+    private HmcUpdateResponse createHmcUpdateResponse(HearingWrapper wrapper) throws InvalidMappingException {
+        HearingRequestPayload hearingPayload = buildHearingPayload(wrapper, referenceDataServiceHolder);
+        String hearingId = getHearingId(wrapper);
+        HmcUpdateResponse updateResponse;
+        String hearingRequestType;
+
+        try {
+            HearingGetResponse getResponse = hmcHearingApiService.getHearingRequest(hearingId);
```

Comment:
I don't think this will work, you'll only have a hearing ID on a successful create request to HMC. The bug arises after we've received a successful response from HMC - if something happens during the ccd update case it will cause Azure to think the entire message failed and cause a retry. 

We want to make sure that in this event we check if a hearing in AWAITING_LISTING exists for the case using the /hearings endpoint (different to /hearing) . If one exists, skip trying to send a new HMC request and go straight to updating the case, else proceed as normal.